pkg/virtctl/vnc/vnc.go

```diff
@@ -70,6 +73,7 @@ func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 			return c.Run(cmd, args)
 		},
 	}
+	cmd.Flags().BoolVar(&proxyOnly, "proxy-only", proxyOnly, "--proxy-only=false: Setting this true will run only the virtctl vnc proxy and show the localhost port where VNC viewers can connect")
```

Comment:
What I would immediately look for is for a way to set the listen port.

Reply:
Added an option for it now

Reply:
:+1